lib/chef/azure/commands/enable.rb

```diff
@@ -132,7 +132,7 @@ def configure_chef_only_once
         Chef::Config[:validation_client_name] =  bootstrap_options['validation_client_name'] if bootstrap_options['validation_client_name']
         template_file = File.expand_path(File.dirname(File.dirname(__FILE__)))
         config[:secret] =  bootstrap_options['secret'] || bootstrap_options['encrypted_data_bag_secret']
-        runlist = @run_list.empty? ? [] : [escape_runlist(@run_list)]
+        runlist = @run_list.empty? ? [] : escape_runlist(@run_list)
         if windows?

```

Comment:
There is a unit testcase existing for 'configure_chef_only_once' method. I'll modify that testcase as a part of this PR https://github.com/chef-partners/azure-chef-extension/pull/16